source/index.rst

```diff
@@ -1,13 +1,13 @@
-Welcome to DecompFrontier wiki
+Welcome to the DecompFrontier Wiki
 ===================================
 
 This wiki contains developer's documentation and analysis of the architecture, protocol
 and formats of the Brave Frontier (v2.19.6.0) game.
 
-.. warning::
+.. Notice::
```

Comment:
This is not a valid sphinx immaterial/restructued text directive, see here https://sphinx-immaterial.readthedocs.io/en/latest/admonitions.html for more information.